nixos/modules/security/apparmor/profiles.nix

```diff
@@ -0,0 +1,315 @@
+    ${etcRule "bash_completion.d"}
+    ${etcRule {path="bash_completion.d"; trail="/*";}}
+    # bash relies on system-wide readline configuration
+    ${etcRule "inputrc"}
+    # bash inspects filesystems at startup
+    # and /etc/mtab is linked to /proc/mounts
+    @{PROC}/mounts
+
+    # run out of /etc/bash.bashrc
+    ${etcRule "DIR_COLORS"}
+  '';
+  "abstractions/cups-client" = ''
+    include ${pkgs.apparmor-profiles}/etc/apparmor.d/abstractions/cpus-client
+    ${etcRule "cups/cups-client.conf"}
+  '';
+  "abstractions/consoles" = ''
+     include ${pkgs.apparmor-profiles}/etc/apparmor.d/abstractions/consoles
+  '';
+  "abstractions/dbus-session-strict" = ''
+    include ${pkgs.apparmor-profiles}/etc/apparmor.d/abstractions/dbus-session-strict
+    ${etcRule "machine-id"}
+  '';
+  "abstractions/dconf" = ''
+    include ${pkgs.apparmor-profiles}/etc/apparmor.d/abstractions/dconf
+    ${etcRule {path="dconf"; trail="/**";}}
+  '';
+  "abstractions/dri-common" = ''
+    include ${pkgs.apparmor-profiles}/etc/apparmor.d/abstractions/dri-common
+    ${etcRule "drirc"}
+  '';
+  # The config.fonts.fontconfig NixOS module adds many files to /etc/fonts/
+  # by symlinking them but without exporting them outside of its NixOS module,
+  # those are therefore added there to this "abstractions/fonts".
+  "abstractions/fonts" = ''
+    include ${pkgs.apparmor-profiles}/etc/apparmor.d/abstractions/fonts
+    ${etcRule {path="fonts"; trail="/**";}}
+  '';
+  "abstractions/gnome" = ''
+    include ${pkgs.apparmor-profiles}/etc/apparmor.d/abstractions/gnome
+    ${etcRule {path="gnome"; trail="/gtkrc*";}}
+    ${etcRule {path="gtk"; trail="/*";}}
+    ${etcRule {path="gtk-2.0"; trail="/*";}}
+    ${etcRule {path="gtk-3.0"; trail="/*";}}
+    ${etcRule "orbitrc"}
+    include <abstractions/fonts>
+    ${etcRule {path="pango"; trail="/*";}}
+    ${etcRule {path="/etc/gnome-vfs-2.0"; trail="/modules/";}}
+    ${etcRule {path="/etc/gnome-vfs-2.0"; trail="/modules/*";}}
+    ${etcRule "papersize"}
+    ${etcRule {path="cups"; trail="/lpoptions";}}
+    ${etcRule {path="gnome"; trail="/defaults.list";}}
+    ${etcRule {path="xdg"; trail="/{,*-}mimeapps.list";}}
+    ${etcRule "xdg/mimeapps.list"}
+  '';
+  "abstractions/kde" = ''
+    include ${pkgs.apparmor-profiles}/etc/apparmor.d/abstractions/kde
+    ${etcRule {path="qt3"; trail="/kstylerc";}}
+    ${etcRule {path="qt3"; trail="/qt_plugins_3.3rc";}}
+    ${etcRule {path="qt3"; trail="/qtrc";}}
+    ${etcRule "kderc"}
+    ${etcRule {path="kde3"; trail="/*";}}
+    ${etcRule "kde4rc"}
+    ${etcRule {path="xdg"; trail="/kdeglobals";}}
+    ${etcRule {path="xdg"; trail="/Trolltech.conf";}}
+  '';
+  "abstractions/kerberosclient" = ''
+    include ${pkgs.apparmor-profiles}/etc/apparmor.d/abstractions/kerberosclient
+    ${etcRule {path="krb5.keytab"; mode="rk";}}
+    ${etcRule "krb5.conf"}
+    ${etcRule "krb5.conf.d"}
+    ${etcRule {path="krb5.conf.d"; trail="/*";}}
+    
```

Comment:
I see CI is complaining that the four spaces on this blank line should be deleted.